flink-cdc-connect/flink-cdc-source-connectors/flink-connector-postgres-cdc/src/main/java/org/apache/flink/cdc/connectors/postgres/source/config/PostgresSourceConfigFactory.java

```diff
@@ -100,7 +102,9 @@ public PostgresSourceConfig create(int subtaskId) {
 
         // The PostgresSource will do snapshot according to its StartupMode.
         // Do not need debezium to do the snapshot work.
-        props.put("snapshot.mode", "never");
+        props.setProperty("snapshot.mode", "never");
+
+        props.setProperty("checkpoint.cycle", String.valueOf(checkpointCycle));
```

Comment:
I don't know what "checkpoint.cycle" does? Debezium's offet commit cycle? Flink cdc have already been responsible for submitting offset, please not let Debezium do it again(turn off)

Reply:
Ahh yes, this was my poor attempt to provide the parameter with properties to the dialect.

I just realized that this is for Debezium, removing it. And I will consider it differently 👍